src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -1742,33 +1742,11 @@ private static int LocateLastFoundByte(Vector<byte> match)
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static int LocateFirstFoundByte(ulong match)
-        {
-            if (Bmi1.X64.IsSupported)
-            {
-                return (int)(Bmi1.X64.TrailingZeroCount(match) >> 3);
```

Comment:
This code was dead as of https://github.com/dotnet/runtime/pull/32371, since it was used only on the `Vector<byte>` fallback for when SSE2 is not available.  Since SSE2 is required for BMI1, this was unreachable.